blueprints/cloud-operations/network-dashboard/variables.tf

```diff
@@ -82,3 +93,8 @@ variable "schedule_cron" {
   description = "Cron format schedule to run the Cloud Function. Default is every 10 minutes."
   default     = "*/10 * * * *"
 }
+
+variable "vpc_connector_name" {
+  description = "Serverless VPC connection name for the Cloud Function"
+  default     = ""
```

Comment:
I seem to remember we already had a variable for this; if that's not the case, could you check cloud run module and see if we can offer the same interface?

BTW default for empty strings is `null` not `""`

Reply:
done